bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/contentfragment/ModelElementsDataSourceServlet.java

```diff
@@ -73,6 +74,8 @@ protected void doGet(@NotNull SlingHttpServletRequest request, @NotNull SlingHtt
         // First try to get the model path from request parameters
         // otherwise determine model path from component resource.
         RequestParameter modelPathRequestParameter = request.getRequestParameter(PARAMETER_AND_PN_MODEL_PATH);
+        String resourcePath = request.getRequestPathInfo().getResourcePath();
+        boolean isOrderBy = StringUtils.endsWith(resourcePath, "/orderBy");
```

Comment:
This requires the node consuming the datasource to be named `orderBy` if I read this correct? We could make this more flexible by allowing an additional property to be supplied to the datasource.